PhysicsTools/PatAlgos/python/slimming/slimmedLowPtElectrons_cff.py

```diff
@@ -4,17 +4,35 @@
 from PhysicsTools.PatAlgos.slimming.lowPtGsfLinks_cfi import lowPtGsfLinks
 from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import lowPtGsfElectronID
 
-# Clone to run ID V2 on patLowPtElectrons
-patLowPtGsfElectronID = lowPtGsfElectronID.clone(
-    electrons='patLowPtElectrons',
-    rho='fixedGridRhoFastjetAll',
-    ModelWeights=cms.vstring(['RecoEgamma/ElectronIdentification/data/LowPtElectrons/LowPtElectrons_ID_2020Sept15.root']),
-    Version=cms.string('V1'),
-)
-
 # Task
 slimmedLowPtElectronsTask = cms.Task(
     lowPtGsfLinks,
-    patLowPtGsfElectronID,
     slimmedLowPtElectrons,
 )
+
+# Modifiers for BParking
+#from Configuration.Eras.Modifier_bParking_cff import bParking
```

Comment:
what's the purpose of this?
either this (and following) commented out code is needed for bParking, in which case it should be uncommented,
or it's not relevant anymore and should be removed

Reply:
Removed.